healthcheck/contrib/django/status_endpoint/tests/test_views.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import connections
+from django.test import override_settings, TestCase
```

Comment:
I101 Imported names are in the wrong order. Should be TestCase, override_settings